koku/api/report/aws/aws_query_handler.py

```diff
@@ -67,7 +67,7 @@ def annotations(self):
         units_fallback = self._mapper._report_type_map.get('units_fallback')
         annotations = {
             'date': self.date_trunc('usage_start'),
-            'units': Coalesce(Value(units_fallback), Concat(self._mapper.units_key, Value('')))
+            'units': Coalesce(Concat(self._mapper.units_key, Value('')), Value(units_fallback))
```

Comment:
New problem here is that this will always return `''` as that's the fallback for self._mapper.units_key in the concat. Sorry. I think we need to get self._mapper.units_key to return a null when it's empty so we can fallback to the units_fallback properly.

Reply:
🤦‍♂️  
I think the change below fixes it.